src/components/Policy/WizardSteps/ReviewStep.tsx

```diff
@@ -21,6 +22,7 @@ const ReviewStep: React.FunctionComponent = () => {
     return (
         <>
             <Form>
+                <Label>{Messages.wizardReview}</Label>
```

Comment:
I think you can use the [Title](https://patternfly-react.surge.sh/documentation/react/components/title) component, IMO the `xl Title` size would be good.

Reply:
Done. Looks better.